build/configuration/pom.xml

```diff
@@ -159,13 +159,15 @@
       <version.jakarta.json>2.0.0</version.jakarta.json>
       <version.jakarta.transaction.legacy>1.3.3</version.jakarta.transaction.legacy>
       <version.jakarta.transaction>2.0.1</version.jakarta.transaction>
-      <version.jakarta.enterprise.cdi>2.0.2</version.jakarta.enterprise.cdi>
+      <version.jakarta.enterprise.cdi.legacy>2.0.2</version.jakarta.enterprise.cdi.legacy>
+      <version.jakarta.enterprise.cdi>3.0.1</version.jakarta.enterprise.cdi>
       <version.jakarta.persistence>3.1.0</version.jakarta.persistence>
       <version.javax.cache>1.1.0</version.javax.cache>
       <version.jboss.logging>3.5.0.Final</version.jboss.logging>
       <version.jboss.marshalling>2.1.1.Final</version.jboss.marshalling>
       <version.jboss.naming>5.0.6.CR1</version.jboss.naming>
-      <version.jboss.narayana>5.13.1.Final</version.jboss.narayana>
+      <version.jboss.narayana.legacy>5.13.1.Final</version.jboss.narayana.legacy>
+      <version.jboss.narayana>6.0.0.Final</version.jboss.narayana>
```

Comment:
why do we need 2 different versions? Naryana is only used for testing.

Reply:
Good spot, I think that was a leftover from an older approach. Removed.